Assets/Editor/BuildTiltBrushPostProcess.cs

```diff
@@ -69,13 +69,13 @@ public void OnPostGenerateGradleAndroidProject(string path)
             );
 #endif
 
-#if ENABLE_BOUNDARYLESS_APP
-            UnityEngine.Debug.Log("Add com.oculus.feature.BOUNDARYLESS_APP");
+#if ENABLE_CONTEXTUAL_BOUNDARYLESS_APP
+            UnityEngine.Debug.Log("Add com.oculus.feature.CONTEXTUAL_BOUNDARYLESS_APP");
             AddOrRemoveTag(doc,
                     androidNamespaceURI,
                     "/manifest",
                     "uses-feature",
-                    "com.oculus.feature.BOUNDARYLESS_APP",
+                    "com.oculus.feature.CONTEXTUAL_BOUNDARYLESS_APP",
```

Comment:
I think this feature/permission may be added automatically if the ovrsettings file is set up correctly: https://developer.oculus.com/documentation/unity/unity-boundaryless/#contextual-boundaryless-implementation 
![image](https://github.com/user-attachments/assets/5ac70d29-e3b0-492b-9dfc-92e870aa8ebd)

Reply:
It seems to work as it is so is this just a cleanup?